doc/source/whatsnew/v1.0.0.rst

```diff
@@ -428,6 +438,8 @@ Optional libraries below the lowest tested version may still work, but are not c
 +-----------------+-----------------+---------+
 | matplotlib      | 2.2.2           |         |
 +-----------------+-----------------+---------+
+| numba           | 0.46.0          |         |
```

Comment:
did you add in install.rst?

Reply:
Yup: https://github.com/pandas-dev/pandas/pull/30151/files#diff-362c4824d3dfaac023ab9a813a70f013R258